core/src/main/java/com/scalar/db/storage/cassandra/CassandraAdmin.java

```diff
@@ -58,7 +59,7 @@ public void createTable(
   @Override
   public void createNamespace(String namespace, Map<String, String> options)
       throws ExecutionException {
-    CreateKeyspace query = SchemaBuilder.createKeyspace(namespace);
+    CreateKeyspace query = SchemaBuilder.createKeyspace(Metadata.quoteIfNecessary(namespace));
```

Comment:
Maybe it's better to use static import.

Reply:
I changed to use static import [commit](https://github.com/scalar-labs/scalardb/pull/441/commits/2e093d1dcbb93f2e1387f73fcb1c4349d52af300)